src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -768,6 +784,14 @@
         <summary>Referenced member.</summary>
       </Comments>
     </Property>
+    <Property Name="ConstrainedToType" Type="ITypeSymbol?" SkipGeneration="true">
```

Comment:
Rather than this convoluted `SkipGeneration` and then proxying through a manually-implemented abstract property, I think it would be much simpler to mark this property `IsAbstract`, and then make a small change to the generator to support an `IsOverride` flag for the derived properties. The only one that would then need to be manually implemented is `IFieldReferenceOperation`, to just return `null`.

Reply:
I think that would require more work with no apparent benefit.

Reply:
It would not require more work: it would require approximately 4 lines of code (3 in the model [here](https://github.com/dotnet/roslyn/blob/main/src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/Model.cs#L102) and one in the writer [here](https://github.com/dotnet/roslyn/blob/main/src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.cs#L630)).

More generally, it is a goal for IOperation to be as generated as possible, to assist in refactorings of the implementation (multiple have happened already in the life of IOperation). Manual implementation is reserved for specific patterns that are not common to IOperation as a whole. This is not such a case: the idea of an abstract base property implemented by either returning a constant value or an auto-prop in derived types is not specific to this part of the hierarchy, and could be reused in the future by new IOperation nodes. Please do this refactoring.